src/components/signUp/Signup.js

```diff
@@ -0,0 +1,146 @@
+import { Button, Container, Grid, Paper } from "@mui/material";
+import React, { useState } from "react";
+import logo from "../../img/logo.png";
+import signUpImg from "../../img/singUp-img.png";
```

Comment:
If you have anything in public:
U can directly use it by `/` without writing public.

Like: `/assets/images/logo.png`

If you want to import that before usage, import like:

`const logo = '/assets/images/logo.png'`